spec/acceptance/z_last_sqlserver_features_spec.rb

```diff
@@ -121,8 +121,8 @@ def ensure_sql_features(features, ensure_val = 'present')
       end
     end
 
-    # TODO: Guard on SQL 2016
-    it "'ADV_SSMS'", :unless => sql_version == '2016', :tier_low => true do
+    # TODO: Guard on SQL 2016 and 2017
+    it "'ADV_SSMS'", :unless => sql_version == '2016' || '2017', :tier_low => true do
```

Comment:
Same here. This line already has two specific versions. A third would start to look cludgy wouldn't it?